OWNERS

```diff
@@ -0,0 +1,10 @@
+reviewers:
+  - alanmeadows
+  - v1k0d3n
+  - intlabs
+  - wilkers-steve
+  - DTadrzak
```

Comment:
remove DTadrzak, as he is not part of this repo ( unless you have suggested otherwise @alanmeadows? )